czkawka_cli/src/main.rs

```diff
@@ -1,395 +1,182 @@
-            yf.print_results();
-
-            yf.get_text_messages().print_messages();
+            ef.print_results();
+            ef.get_text_messages().print_messages();
         }
-        "--t" => {
-            let mut tf = temporary::Temporary::new();
-
-            if ArgumentsPair::has_command(&arguments, "-i") {
-                tf.set_included_directory(ArgumentsPair::get_argument(&arguments, "-i", false));
-            } else {
-                println!("FATAL ERROR: Parameter -i with set of included files is required.");
-                process::exit(1);
-            }
-
-            if ArgumentsPair::has_command(&arguments, "-e") {
-                tf.set_excluded_directory(ArgumentsPair::get_argument(&arguments, "-e", false));
-            }
-
-            if ArgumentsPair::has_command(&arguments, "-k") {
-                tf.set_excluded_items(ArgumentsPair::get_argument(&arguments, "-k", false));
-            }
-
-            if ArgumentsPair::has_command(&arguments, "-o") {
-                tf.set_recursive_search(false);
-            }
-
-            if ArgumentsPair::has_command(&arguments, "-delete") {
+        Commands::Temporary {
+            directories,
+            excluded_directories,
+            excluded_items,
+            delete_files,
+            file_to_save,
+            not_recursive,
+        } => {
+            let mut tf = Temporary::new();
+
+            tf.set_included_directory(path_list_to_str(directories.directories));
+            tf.set_excluded_directory(path_list_to_str(excluded_directories.excluded_directories));
+            tf.set_excluded_items(path_list_to_str(excluded_items.excluded_items));
+            tf.set_recursive_search(!not_recursive.not_recursive);
+
+            if delete_files {
                 tf.set_delete_method(temporary::DeleteMethod::Delete);
             }
 
             tf.find_temporary_files();
 
-            #[allow(clippy::collapsible_if)]
-            if ArgumentsPair::has_command(&arguments, "-f") {
-                let file_name = match ArgumentsPair::get_argument(&arguments, "-f", true).as_str() {
-                    "" => "results.txt".to_string(),
-                    t => t.to_string(),
-                };
-                if !tf.save_results_to_file(file_name.as_str()) {
+            if let Some(file_name) = file_to_save.file_name() {
+                if !tf.save_results_to_file(file_name) {
                     tf.get_text_messages().print_messages();
                     process::exit(1);
                 }
             }
 
             #[cfg(not(debug_assertions))] // This will show too much probably unnecessary data to debug, comment line only if needed
             tf.print_results();
-
             tf.get_text_messages().print_messages();
         }
-        "--version" | "--v" => {
-            println!("Czkawka CLI {}", CZKAWKA_VERSION);
-            process::exit(0);
-        }
```

Comment:
Was the `--version` intentionally removed?
Is there another way to query for it?

Reply:
It was probably left out by a mistake.